src/lib/textToCad.ts

```diff
@@ -0,0 +1,22 @@
+import { Models } from '@kittycad/lib'
+import { VITE_KC_API_BASE_URL } from 'env'
+
+export async function submitTextToCadPrompt(
+  prompt: string
+): Promise<Models['TextToCad_type'] | Error> {
+  const body: Models['TextToCadCreateBody_type'] = { prompt }
+  const response = await fetch(
+    VITE_KC_API_BASE_URL + '/ai/text-to-cad/gltf?kcl=true',
+    {
+      method: 'POST',
+      body: JSON.stringify(body),
```

Comment:
this will need to be authed, I didn't look into how we are doing that elsewhere, but just keep it in mind.

Reply:
probably `credentials: true` or some shit